core/api_views.py

```diff
@@ -24,7 +24,7 @@ def get_serializer_class(self: "CommunityDetailAPIView") -> CommunitySerializer:
 
 
 class PostAPIListView(ListAPIView):
-    queryset = Post.objects.exclude(community__privacy=Community.PRIVATE)
+    queryset = Post.objects.exclude(community__privacy=Community.PRIVATE).order_by("-updated_at")
```

Comment:
I am thinking this should be not -updated_at but by id no? Beceause if we change order -updated here is what happens
- I write post A
- You write post B
Order on page is 
A
B

Now I edit post A

Order will be 
B
A

Am I correct? What are your thoughts?

Reply:
I think you are right. Is order of django ids something we can rely on?
There is `created_at` field - that would work the same way, but be more explicit. What do you think?

Reply:
Consider created_at just for displaying purposes - to show date when post are created. 
Id is auto increment - new post, new id, so using ordering by id is something we can rely on. If we use created_at for ordering, we have the same effect, only more DB load. So let's stick to ids.